tests/unit/panel/panel_core.js

```diff
@@ -472,14 +472,14 @@ QUnit.asyncTest( "overlay panel should not call getWrapper", function( assert )
 		},
 		function() {
 			assert.deepEqual( count, 0, "getWrapper not called on close" );
-			$.mobile.changePage( "#page2" );
+			$( ".ui-pagecontainer" ).pagecontainer( "change", "#page2" );
```

Comment:
I guess this is OK for testing purposes since we control the content and we know there is only one pagecontainer. Still, maybe $.mobile.pagecontainers.active is in order?